main.go

```diff
@@ -50,11 +51,19 @@ func (s *server) makeGenerateQRCodeHandler() http.HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request) {
 		text := s.text
 		logoPath := s.logoPath
+		qrSize := 256
```

Comment:
I suggest that you put all default values as constants at the top of the file

```go
const (
     defaultPort = 8080
     defaultSize = 256    
)
```

and not "hide" them in the middle of the code. (I also made port an int (actually an untyped number constant) here).